src/templates/docPage.js

```diff
@@ -14,14 +15,30 @@ import {
 import MachineTranslationCallout from '../components/MachineTranslationCallout';
 import SEO from '../components/SEO';
 import GithubSlugger from 'github-slugger';
-import { parseHeading } from '../../plugins/gatsby-remark-custom-heading-ids/utils/heading';
```

Comment:
is this getting used anywhere else or can we delete it from the repo?

Reply:
looks like it's still used in [TableOfContents.js](https://github.com/newrelic/docs-website/blob/develop/src/components/TableOfContents.js#L9) and [serialize-mdx.js](https://github.com/newrelic/docs-website/blob/develop/scripts/actions/serialize-mdx.js#L12)